src/internal/common/dttm/dttm.go

```diff
@@ -21,23 +21,23 @@ const (
 	// non-json cli outputs.
 	TabularOutput TimeFormat = "2006-01-02T15:04:05Z"
 
-	// LegacyTime is used in /exchange/service_restore to comply with certain
+	// Legacy is used in /exchange/service_restore to comply with certain
 	// graphAPI time format requirements.
-	LegacyTime TimeFormat = time.RFC3339
+	Legacy TimeFormat = time.RFC3339
 
-	// SimpleDateTime is the default value appended to the root restoration folder name.
-	SimpleDateTime TimeFormat = "02-Jan-2006_15:04:05"
-	// SimpleDateTimeOneDrive modifies SimpleDateTimeFormat to comply with onedrive folder
+	// HumanReadable is the default value appended to the root restoration folder name.
+	HumanReadable TimeFormat = "02-Jan-2006_15:04:05"
+	// HumanReadableDriveItem modifies SimpleDateTimeFormat to comply with onedrive folder
 	// restrictions: primarily swapping `-` instead of `:` which is a reserved character.
-	SimpleDateTimeOneDrive TimeFormat = "02-Jan-2006_15-04-05"
+	HumanReadableDriveItem TimeFormat = "02-Jan-2006_15-04-05"
 
 	// m365 will remove the :00 second suffix on folder names, resulting in the following formats.
-	ClippedSimple         TimeFormat = "02-Jan-2006_15:04"
-	ClippedSimpleOneDrive TimeFormat = "02-Jan-2006_15-04"
+	ClippedHuman          TimeFormat = "02-Jan-2006_15:04"
```

Comment:
`ClippedHuman` sounds like a person who has undergone robotic enhancements, lol. I was going to suggest `TruncatedHuman` but thats not any better.